packages/api/src/external/carequality/document/shared.ts

```diff
@@ -87,3 +92,60 @@ const generateCQFHIRContent = (
 
   return [cqFHIRContent];
 };
+
+function generateCQFHIRContained(authorInstitution: string | undefined): Organization | undefined {
```

Comment:
If this is "contained" (no specific resource type), can we name the parameter accordingly, so it's more generic, please?

OTOH, looking below it seems to be tied to organization, so maybe just rename it to be `mapToContainedOrganization`? I intentionally used "map" instead of "generate" because we're mapping a composite ID into a FHIR org, so there goes a suggestion. 😄